app/helpers/buyers_helpers.py

```diff
@@ -99,3 +99,90 @@ def allowed_email_domain(current_user_id, brief, data_api_client=None):
     email_domain = current_user.get('email_address').split('@')[-1]
     brief_user_email = brief.get('users')[0].get('emailAddress').split('@')[-1]
     return email_domain == brief_user_email
+
+
+def remove_non_cascasde_fields(brief, section, current_question_id, update_data=None):
```

Comment:
Looks like there's a typo in the function name and associated references.

Reply:
fixed